data/assets/scene/solarsystem/heliosphere/bastille_day/fieldlines.asset

```diff
@@ -0,0 +1,92 @@
+local assetHelper = asset.require('util/asset_helper')
+local heliosphereTransforms = asset.require('scene/solarsystem/sun/transforms_heliosphere')
+local propertyHelper = asset.require('../../../../util/property_helper')
+
+local fieldlinesDirectory = asset.syncedResource({
+  Name = "Bastille Day MAS Fieldlines",
+  Type = "HttpSynchronization",
+  Identifier = "bastille_day_mas_fieldlines",
+  Version = 1
+})
+
+local sunRadius = 6.957E8
+
+-- Fieldlies from binaries
+local fieldlines = {
+  Identifier = "MAS-MHD-Fieldlines",
+  Parent = heliosphereTransforms.HEEQ180ReferenceFrame.Identifier,
+  Transform = {
+    Scale = {
+      Type = "StaticScale",
+      Scale = sunRadius
+    },
+    Rotation = {
+    -- This is a rotation matrix to go from Carrington longitude referens frame to HEEQ180
+    -- which is what the renderables parent transforms.HEEQ180ReferenceFrame. Identifier is
+    -- referring to. At the reference time, MAS_seq = 0, 2000-07-14T08:33:37.105 the Carrington
+    -- longitude was 309.3 degrees.
+    -- Difference from HEEQ => 360-309.3=50.7
+    -- (or 0-309.3 = -309.3 However this leads to the same rotation matrix in the end)
+    -- Since OpenSpace supports HEEQ180 and not HEEQ, 180 was added or subtracted
+    -- => a1 = -129.3 and a2 = 230.7
+    -- Rotation matrix: (cos a, -sin a, 0)(sin a, cos a, 0)(0, 0, 1) leads to the result.
+    Type = "FixedRotation",
```

Comment:
This line and the comments above should be indented one step (two whitespace) more